packages/fluentui/react-northstar/src/utils/positioner/Popper.tsx

```diff
@@ -182,11 +281,21 @@ export const Popper: React.FunctionComponent<PopperProps> = props => {
           phase: 'afterWrite' as PopperJs.ModifierPhases,
           fn: handleUpdate,
         },
+        popperInitialPositionFix.modifier,
       ].filter(Boolean),
       onFirstUpdate: state => handleUpdate({ state }),
     };
 
     popperInstanceRef.current = PopperJs.createPopper(reference, contentRef.current, options);
+    const originalForceUpdate = popperInstanceRef.current.forceUpdate;
```

Comment:
```suggestion
    positionPatch(popperInstanceRef.current);
```

My original proposal was to move the whole logic there to our hook